src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -93,4 +99,66 @@ public DatasetInfoDto retrieveDataset(
       @ApiParam(value = "id of the dataset", required = true) @PathVariable("id") String datasetId) {
     return reportService.getReport(datasetId);
   }
+
+
+  /**
+   * Get all available countries that can be used.
+   * <p>The list is retrieved based on an internal enum</p>
+   *
+   * @return The list of countries available
+   */
+  @ApiOperation("Get data of all available countries")
+  @ApiResponses({
+      @ApiResponse(code = 200, message = MESSAGE_FOR_RETRIEVE_DATASET, response = Object.class)
+  })
+  @GetMapping(value = "dataset/countries", produces = APPLICATION_JSON_VALUE)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public List<CountryModel> getAllCountries() {
+    return Arrays.stream(Country.values()).map(CountryModel::new).collect(Collectors.toList());
+  }
+
+  /**
+   * Get all available languages that can be used.
+   * <p>The list is retrieved based on an internal enum</p>
+   *
+   * @return The list of countries that are available
+   */
+  @ApiOperation("Get data of all available languages")
+  @ApiResponses({
+      @ApiResponse(code = 200, message = MESSAGE_FOR_RETRIEVE_DATASET, response = Object.class)
+  })
+  @GetMapping(value = "dataset/languages", produces = APPLICATION_JSON_VALUE)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public List<LanguageModel> getDatasetsLanguages(){
+    return Language.getLanguageListSortedByName().stream().map(LanguageModel::new)
+        .collect(Collectors.toList());
+  }
+
+  private static class CountryModel {
```

Comment:
I can see this is basically a copy from metis-framework.
We've decided for any object that is for viewing to suffix it with "View" instead of "Model" and use "Model" for db objects.
I know it's metis-framework this is not updated yet, but perhaps here we can update already

Reply:
So if I understood correctly, instead of `CountryModel` you say to change it to `CountryView`, correct?
If that is it, then I will updated it